cfme/containers/image.py

```diff
@@ -22,9 +22,9 @@
 
 class Image(Taggable, SummaryMixin, Navigatable, PolicyProfileAssignable):
 
-    def __init__(self, name, tag, provider, appliance=None):
+    def __init__(self, name, id_, provider, appliance=None):
```

Comment:
honestly, I don't like such parameter names. I'd advice you to use something like image_id instead